launchable/commands/record/tests.py

```diff
@@ -289,5 +289,14 @@ def get_record_start_at(token: str, org: str, workspace: str, build_name: Option
         # to avoid stop report command
         return datetime.datetime.now()
 
+    return parse_launchable_timeformat(res.json()["createdAt"])
+
+
+def parse_launchable_timeformat(t: str) -> datetime.datetime:
     # e.g) "2021-04-01T09:35:47.934+00:00"
-    return datetime.datetime.strptime(res.json()["createdAt"], "%Y-%m-%dT%H:%M:%S.%f%z")
+    try:
+        return datetime.datetime.strptime(t, "%Y-%m-%dT%H:%M:%S.%f%z")
```

Comment:
I found the Python official document that recommends `python-dateutil` module to parsing ISO8601 https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat . Of course, there is a trade-off between increasing dependency and reusability. You can decide it make sense or not.

Reply:
Thank you for your advice! I'll add `python-dateutil` and use it